test/specs/trailApexReplayDebugger.e2e.ts

```diff
@@ -0,0 +1,209 @@
+  let prompt: QuickOpenBox | InputBox;
+  let testSetup: TestSetup;
+  const fiveMinutes = 5 * 60;
+
+  step('Set up the testing environment', async () => {
+    testSetup = new TestSetup('TrailApexReplayDebugger', false);
+    await testSetup.setUp();
+
+    // Create Apex class AccountService
+    await utilities.createApexClassWithBugs();
+
+    // Push source to org
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Push Source to Default Org and Override Conflicts', 5);
+  });
+
+  step('Run Apex Tests', async () => {
+    // Run SFDX: Run Apex tests.
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Clear Output', 1);
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Run Apex Tests', 1);
+
+    // Select the "AccountServiceTest" file
+    await prompt.selectQuickPick('AccountServiceTest');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests', fiveMinutes);
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Listening for streaming state changes...', fiveMinutes);
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Processing test run', fiveMinutes, false);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Run Apex Tests successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+
+    // Verify test results are listed on vscode's Output section
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Apex', '=== Test Results', 10);
+    expect(outputPanelText).not.toBeUndefined();
+    expect(outputPanelText).toContain('Assertion Failed: incorrect ticker symbol');
+    expect(outputPanelText).toContain('Expected: CRM, Actual: SFDC');
+  });
+
+  step('Set Breakpoints and Checkpoints', async () => {
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+    const textEditor = await editorView.openEditor('AccountService.cls') as TextEditor;
+    await textEditor.moveCursor(8, 5);
+
+    // Run SFDX: Toggle Checkpoint.
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Toggle Checkpoint', 1);
+
+    // Calling SFDX: Update Checkpoints in Org fails if the org has been recently created, 
+    // it does not complete the 6 steps but only 4.
+    // Reloading the window forces the extensions to be reloaded and this seems to fix the issue.
+    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Reload Window', 10);
+    await utilities.pause(10);
+
+    // Run SFDX: Update Checkpoints in Org.
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Update Checkpoints in Org', 2);
```

Comment:
The pause here at line 81 needs to be a lot longer.  Maybe 10 seconds?  on my machine, updating the checkpoints takes quite a while.